src/services/deleteTokenLogoutService.js

```diff
@@ -19,11 +21,12 @@ angular.module('openshiftCommonServices')
 
         // Lazily get the data service. Can't explicitly depend on it or we get circular dependencies.
         var DataService = $injector.get('DataService');
+        var oAuthAccessTokensVersion = $injector.get('APIService').getPreferredVersion('oauthaccesstokens');
```

Comment:
Nit: I'd find this hard to read as one statement. I'd prefer something like...

```js
// Lazily get the API service and data service. Can't explicitly depend on them or we get circular dependencies.
var APIService = $injector.get('APIService');
var DataService = $injector.get('DataService');

var oAuthAccessTokensVersion = APIService.getPreferredVersion('oauthaccesstokens');
```

Reply:
I'm fine with that. updating.